vars/artifactSetVersion.groovy

```diff
@@ -177,8 +177,36 @@ void call(Map parameters = [:], Closure body = null) {
                 error "[${STEP_NAME}]git commit and tag failed: ${e}"
             }
 
-            sshagent([config.gitSshKeyCredentialsId]) {
-                sh "git push ${config.gitSshUrl} ${config.tagPrefix}${newVersion}"
+            if(config.gitSshUrl) {
+                sshagent([config.gitSshKeyCredentialsId]) {
+                    sh "git push ${config.gitSshUrl} ${config.tagPrefix}${newVersion}"
+                }
+            } else if(config.gitHttpsUrl) {
```

Comment:
`gitSshUrl` is a mandatory parameter of the step, thus we will never reach the http branch of the condition

Reply:
I would suggest to make the decision rather on availability of the `gitSshKeyCredentialsId`

Reply:
maybe the parameter is not mandatory with that change here. Either sshUrl or https url needs to be provided.

Reply:
the urls are generated based on the checked out repo ...

Thus, there will always be both available ...